python/artm/model.py

```diff
@@ -398,7 +386,6 @@ def fit_online(self, batch_vectorizer=None, tau0=1024.0, kappa=0.7,
                                             nwt='nwt_hat',
                                             regularizer_name=theta_reg_name,
                                             regularizer_tau=theta_reg_tau,
-                                            num_inner_iterations=self._num_document_passes,
```

Comment:
Maybe I don't understand... but where the num_document_passes parameter in fit_online()? I think, you need to add it in the method parameters, in the master_component.process_batches() call, and describe it in the docstring.